### ISSUE TITLE
Copyright claim

### ISSUE THREAD

**Issue by bowlofeggs** [fedora-infra/pdc-updater#65]
While reviewing https://github.com/fedora-infra/pdc-updater/pull/63 I noticed that this project doesn't seem to make a copyright claim. I think the recommended practice is to put a copyright header on each file (at least, I feel like I remember the FSF making that recommendation), though I've also seen precedent for making a single COPYRIGHT file in the root of the repo. I don't think it's that important how it's done, but it probably is important to claim copyright on this code in some fashion.

**Comment by tyll** [comment]
good point, I believe this is also mentioned in the licensing guidelines, in the gpl howto and best practice.
